manifests/platform.yaml

```
@@ -244,7 +244,7 @@ spec:
     namespace: argo
   - name: cray-nls
     source: csm-algol60
-    version: 1.4.49
+    version: 1.4.51
```

The latest version that we want is actually 1.4.52.

```suggestion
    version: 1.4.52
```